docs/source/index.md

```diff
@@ -155,7 +155,7 @@ See [Google Scholar](https://scholar.google.de/scholar?oi=bibs&hl=en&authuser=1&
 
 installation
 learning
-Examples <https://docs.pymc.io/projects/examples/en/latest/>
+Examples <nb:index>
```

Comment:
this one is not working like this :/. The preview is missing examples from the navbar. Can you undo the change and add it to the list of excluded files?

Reply:
Done 😄